app/services/idv/attempter.rb

```diff
@@ -23,7 +27,7 @@ def exceeded?
       return false if window_expired?
 
       # too many attempts in the window
-      attempts >= idv_max_attempts && !window_expired?
+      attempts >= self.class.idv_max_attempts && !window_expired?
```

Comment:
To make it public. It's not instance-specific (it's class-wide configuration) and having it as a class method allowed the specs and other places to use it cleanly.